### DISCUSSION TITLE
Using wireless protocol to add wireless functionality to non Simagic bases

### DISCUSSION THREAD

**Discussion by KitaCanada** [Kegetys/simagic-arduino-poc#1] [Q&A]
What do you think the feasibility of doing the opposite and adding wireless connectivity to non Simagic bases with a Simagic wheel? Another question that would be a little more difficult would be what do you think the feasibility of using that wheel input and converting it into a can input for an ecu or something to allow Simagic wheels to wirelessly connect to the car and do car functions?

**Comment by Kegetys** [comment]
I haven't tried it other than accidentally receiving some packets from the rim, but it should work. Only issue is perhaps if the rim expects to receive the ACK packets from the base, a genuine nRF24L01 chip may be required for that so the rim doesn't bombard you with repeated packets. The aliexpress QR comes with the base-side connector as well as mentioned, so you can power the rim with that.

As long as you are able to receive the packets from the rim you can trigger anything you want with them of course.

**Reply by Anguiano76** [reply]
> I haven't tried it other than accidentally receiving some packets from the rim, but it should work. Only issue is perhaps if the rim expects to receive the ACK packets from the base, a genuine nRF24L01 chip may be required for that so the rim doesn't bombard you with repeated packets. The aliexpress QR comes with the base-side connector as well as mentioned, so you can power the rim with that.
> 
> As long as you are able to receive the packets from the rim you can trigger anything you want with them of course.

Hello.

Could you give us some clues as to where to look to configure and connect buttons? I managed to connect perfectly with SimPro Manager, but I get two clutch axes and two buttons and I don't even know where in the code to configure buttons, or where to connect them.

I would appreciate some light.

Greetings and thanks for the project
